manticore/platforms/linux.py

```diff
@@ -1043,13 +1043,13 @@ def set_entry(self, entryPC):
         self.current.PC = elf_entry
         logger.debug(f"Entry point updated: {elf_entry:016x}")
 
-    def load(self, filename: str, env) -> None:
+    def load(self, filename: str, env_list: List) -> None:
```

Comment:
Minor nit: Isn't the `_list` suffix superfluous now that there is a type hint?

Reply:
I renamed the variable to address a type hint error later in the code -- on line 1063, `env` (the previous name) is reassigned to a different type, which `mypy` doesn't like.

An alternative, I suppose, would be to keep the `env` parameter named the same, and rename the variable in the body instead.  Let me know if you want that.

Reply:
Ahh okay. Yeah, I'm not sure I like how `mypy` handles that. I think how it is here is okay. 

Do you have any input @ehennenfent ?

Reply:
Yeah mypy is pretty strict: it doesn't permit a variable to be reassigned with a different type.  So it is.

Reply:
I was grumpy about this at first but have now been converted to the cult of "don't reuse variable for data of different types" 

https://twitter.com/conectado2/status/1248032863238393857